tests/testing.py

```diff
@@ -1267,3 +1268,31 @@ def f(u, v, arg, kwarg=-1.0):
 
     assert 0.0 == pytest.approx(np.linalg.norm(coordinates - coordinates_ref), 1e-12)
     assert np.linalg.norm(connectivity - connectivity_ref) == 0
+
+
+def test_import_fluent_geometry():
+    """ 
+    Test if the simplest comand to import a fluent geometry works without error. 
+    """
+    
+    fluent_geometry= os.path.join(testing_input_geometry,"fluent_aneurysm.msh")
+    cubit = CubitPy()
+    cubit.Import_Fluent_Geometry(fluent_geometry)
+    assert False == cubit.was_last_cmd_undoable()
+
+
+def test_import_fluent_geometry_feature_angle():
```

Comment:
Why the need for two tests here? Isn't the first one exactly contained in this one?

Reply:
Nope, the angle decides, how surfaces are reconstructed. In the first case, you obtain approx. 7 faces (some of them are pretty small), whereas in the second case you end up with the desired 4. If you want to reduce the number of test cases, we can delete the first test case.